docs/dev/copyright_license.md

```diff
@@ -0,0 +1,168 @@
+<!---
+(C) copyright 2019 SuperDARN Canada, University of Saskatchewan 
+(C) copyright 2021 The University Centre in Svalbard (UNIS)
+
+authors: Marina Schmidt, SuperDARN Canada
+         Emma Bland, UNIS
+
+pyDARN is under the LGPL v3 license found in the root directory LICENSE.md 
+Everyone is permitted to copy and distribute verbatim copies of this license 
+document, but changing it is not allowed.
+
+This version of the GNU Lesser General Public License incorporates the terms
+and conditions of version 3 of the GNU Lesser General Public License, supplemented by
+the additional permissions listed below.
+
+       
+Modifications:
+          Emma Bland, UNIS, 2021-02-12 : added background information about LGPL, why copyright information is required, FAQ, and pyDARN license history 
+          Marina Schmidt, University of Saskatchewan: added LGPL license for pyDARN
+-->
+
+
+# Licensing of the pyDARN
```

Comment:
Should this be "the pyDARN library" or just "pyDARN"?

Reply:
``` suggestion
Licensing of pyDARN
```

pyDARN